g-sgov/g-sgov-aktuality/g-sgov-aktuality-glosář-MaNe.ttl

```diff
@@ -68,6 +68,8 @@ g-sgov-aktuality-pojem:má-téma
         a       <http://www.w3.org/2004/02/skos/core#Concept> ;
         <http://www.w3.org/2004/02/skos/core#broader>
                 <https://slovník.gov.cz/základní/pojem/vztah> , <https://slovník.gov.cz/základní/pojem/typ-vztahu> ;
+        <http://www.w3.org/2004/02/skos/core#hiddenLabel>
```

Comment:
Proč je to hidden label a ne alt label?

Reply:
Mal som v tychto pripadoch pocit, ze hidden label je lepsi, no presvedcil si ma o opaku :), update-ol som to poslednym commitom.